text/getting_started.tex

```diff
@@ -81,30 +81,33 @@ \section{Getting the RTOS source code}
 \end{console}
 
 This command will create a single instance of the container with the current directory mounted as \file{/home/cheriot/cheriot-rtos}.
-This creates an ephemeral instance, you can create persistent instances with \command{docker create}.
+This creates an ephemeral instance. You can create persistent instances with \command{docker create} instead of docker run.
 
 In the dev container, all of the CHERIoT tools are installed in \file{/cheriot-tools/bin/}.
 
 \begin{note}
 	The dev container also exists to support \keyword{GitHub Code Spaces}.
 	These run a Visual Studio Code instance in a browser, attached to a dev container deployed in an Azure VM.
-	If you create a Code Space from the CHERIoT RTOS repository, it will be set up with everything that you need to develop for CHERIoT, in the browser.
+	If you create a Code Space from the CHERIoT RTOS repository, it will be set up with everything that you need to develop for CHERIoT in the browser.
 	GitHub Code Spaces are a good way to start playing with CHERIoT, but the free tier is limited to 120 CPU hours (60 hours on the smallest VM tier) and so you will probably want to install the toolchain locally for serious development.
 \end{note}
 
 \section{Setting up a development environment}
 
 Having a copy of the RTOS software does not enable you to build it.
-You will also need a \keyword{toolchain}, a compiler, linker, and other associated tools that can take source code and turn it into a firmware image that can run on a device.
+You will also need a \keyword{toolchain}: a compiler, linker, and other associated tools that can take source code and turn it into a firmware image that can run on a device.
+\amanda{I think.  If the toolchain doesn't contain the compiler, linker, etc then feel free to ignore this change.}
 If are using the dev container, these tools are all installed for you.
 
-The CHERIoT toolchain is based on LLVM, which used to stand for 'Low Level Virtual Machine' until it became clear that none of those words actually applied to the project and is now just a name.
+The CHERIoT toolchain is based on LLVM, which used to stand for 'Low Level Virtual Machine' until it became clear that none of those words actually applied to the project and it is now just a name.
+\amanda{This sentence made me laugh very hard.  I love it!}
 LLVM is a generic set of building blocks for writing compilers, structured around the \keyword{LLVM Intermediate Representation} (LLVM IR).
-It includes a mature C/C++ front end, a component that transforms C and C++ (and Objective-C) into LLVM IR.
+It includes a mature C/C++ front end and a component that transforms C and C++ (and Objective-C) into LLVM IR.
```

Comment:
This is not correct.  The front end *is* the component that transforms C into LLVM IR.